From 66a7969f53c198865cf09e0690aa17100797fe03 Mon Sep 17 00:00:00 2001
From: Smadar Fuks <smadarf@marvell.com>
Date: Sun, 23 Aug 2020 14:42:14 -0400
Subject: [PATCH 0700/1921] octeontx2-af: Fix REE warnings for no previous
 prototype

These warnings were due to missing static declarations

Change-Id: Ic41cc057ea97e18b7be23e24aaa0e2131a0cec89
Signed-off-by: Smadar Fuks <smadarf@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/34939
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_ree.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_ree.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_ree.c
index 78d5367efb22..6eb80ffd4c12 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_ree.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_ree.c
@@ -985,7 +985,7 @@ static irqreturn_t rvu_ree_af_aq_intr_handler(int irq, void *ptr)
 	return IRQ_HANDLED;
 }
 
-void rvu_ree_unregister_interrupts_block(struct rvu *rvu, int blkaddr)
+static void rvu_ree_unregister_interrupts_block(struct rvu *rvu, int blkaddr)
 {
 	int i, offs;
 
@@ -1038,7 +1038,7 @@ static int rvu_ree_af_request_irq(struct rvu_block *block,
 	return rvu->irq_allocated[offset];
 }
 
-int rvu_ree_register_interrupts_block(struct rvu *rvu, int blkaddr)
+static int rvu_ree_register_interrupts_block(struct rvu *rvu, int blkaddr)
 {
 	struct rvu_hwinfo *hw = rvu->hw;
 	struct rvu_block *block;
@@ -1195,7 +1195,7 @@ int rvu_ree_init(struct rvu *rvu)
 	return rvu_ree_init_block(rvu, BLKADDR_REE1);
 }
 
-void rvu_ree_freemem_block(struct rvu *rvu, int blkaddr, int blkid)
+static void rvu_ree_freemem_block(struct rvu *rvu, int blkaddr, int blkid)
 {
 	struct rvu_hwinfo *hw = rvu->hw;
 	struct rvu_block *block;
-- 
2.31.1

